packages/tradershub/src/components/Timeline/Timeline.classnames.ts

```diff
@@ -3,33 +3,33 @@ import { cva } from 'class-variance-authority';
 const stepperBaseStyles = 'z-10 box-border flex h-800 w-800 items-center rounded-pill outline outline-2';
 
 export const stepperVariants = cva(stepperBaseStyles, {
-    variants: {
-        isActive: {
-            true: 'transition-all delay-700 duration-700 ease-out  outline-solid-coral-700',
-            false: 'outline-solid-grey-5',
-        },
-        isFilled: {
-            true: '',
-        },
-    },
     compoundVariants: [
         {
+            class: 'bg-solid-grey-5',
             isActive: false,
             isFilled: true,
-            class: 'bg-solid-grey-5',
         },
         {
+            class: 'bg-solid-coral-700',
             isActive: true,
             isFilled: true,
-            class: 'bg-solid-coral-700',
         },
     ],
+    variants: {
+        isActive: {
+            false: 'outline-solid-grey-5',
+            true: 'transition-all delay-700 duration-700 ease-out  outline-solid-coral-700',
+        },
+        isFilled: {
+            true: '',
+        },
```

Comment:
what's this for?

Reply:
<img width="265" alt="Screenshot 2024-01-30 at 2 58 12 PM" src="https://github.com/binary-com/deriv-app/assets/129842155/7810fbce-87d7-4af2-8817-393c53fa657c">
This was implemented by @likhith-deriv to have this filled

Reply:
what happens if it's false?

Reply:
The styles are not pretty accurate. We'll do an improvement PR for this component